MMA Removal Utility/README.md

```diff
@@ -238,8 +239,82 @@ let virtualMachinesWithBothExtensions = virtualMachines
 virtualMachinesWithBothExtensions
 ```
 
-### **8. How I can get list of all the VMs which MMA agent present?**
-VMs and Extensions inventory is collected in Log Analytics workspace. To list all the VMs which have both MMA agent present, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
+For Virtual Machines Scale Set (VMSS):
+
+``` KQL
+let timeago = timespan(7d);
+let virtualMachines = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceId contains "microsoft.compute/virtualmachinescalesets"
```

Comment:
Lets also make sure we test all the queries once.

Reply:
All queries are tested